packages/electrode-archetype-react-app-dev/config/archetype.js

```diff
@@ -16,9 +16,10 @@ module.exports = {
   devPkg,
   devRequire,
   webpack: Object.assign({}, {
-    devHostname: process.env.WEBPACK_HOST || "localhost",
+    devHostname: process.env.WEBPACK_DEV_HOST || "localhost",
     devPort: utils.getInt(process.env.WEBPACK_DEV_PORT, 2992),
     testPort: utils.getInt(process.env.WEBPACK_TEST_PORT, 3001),
+    devProtocol: process.env.WEBPACK_DEV_HTTPS,
```

Comment:
@jchip Original idea is followed by the pattern like devPort and devHostName, will `protocol` a good name instead? Any recommendations?